src/uri/uauthority.rs

```diff
@@ -13,11 +13,8 @@
 
 use bytes::BufMut;
 
-use crate::uprotocol::uri::uauthority::Number;
-use crate::uprotocol::UAuthority;
-
-pub use crate::proto::uprotocol::uuri::SerializationError;
-use crate::uri::validator::ValidationError;
+use crate::errors::{SerializationError, ValidationError};
```

Comment:
FMPOV this needs to be `pub use crate ...`. Otherwise, no code outside of this crate will be able to handle an error produced by the `TryFrom` trait implementations because the `error` module has crate visibility only ...

Reply:
Done